docs/src/pages/components/dividers/ListDividers.tsx

```diff
@@ -1,25 +1,20 @@
 import * as React from 'react';
-import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import Divider from '@material-ui/core/Divider';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      width: '100%',
-      maxWidth: 360,
-      backgroundColor: theme.palette.background.paper,
-    },
-  }),
-);
-
 export default function ListDividers() {
-  const classes = useStyles();
-
   return (
-    <List component="nav" className={classes.root} aria-label="mailbox folders">
+    <List
+      sx={{
+        width: '100%',
+        maxWidth: 360,
+        backgroundColor: 'background.paper',
```

Comment:
We have used this shorthand so far,

```suggestion
        bgcolor: 'background.paper',
```

Reply:
Resolved